data/avro-schema/src/main/resources/avro/net/corda/data/membership/common/RegistrationStatus.avsc

```diff
@@ -4,7 +4,8 @@
   "namespace": "net.corda.data.membership.common",
   "doc": "Registration request status.",
   "symbols": [
-    "NEW",
+    "SUBMITTED",
+    "PENDING_MGM_NETWORK_ACCESS",
```

Comment:
Is it worth to keep `NEW`, instead of `SUBMITTED`, to avoid misleading people into thinking it's been submitted to the MGM?

Reply:
Or alternatively `ACCEPTED` instead of `NEW`

Reply:
the issue I see with `ACCEPTED` is people might be thinking "accepted by the MGM" (not the cluster). As always, naming is the biggest problem..